src/toil/cwl/cwltoil.py

```diff
@@ -37,7 +37,24 @@
 from collections.abc import Iterator, Mapping, MutableMapping, MutableSequence
 from tempfile import NamedTemporaryFile, TemporaryFile, gettempdir
 from threading import Thread
-from typing import IO, Any, Callable, Literal, Optional, TextIO, TypeVar, Union, cast
+from typing import (
+    IO,
+    Any,
+    Callable,
+    Dict,
+    Iterator,
+    List,
```

Comment:
I think we don't need `Dict` and `List` anymore because we can use `dict` and `list` now.